docker/types/containers.py

```diff
@@ -473,6 +473,9 @@ def __init__(self, version, binds=None, port_bindings=None,
 
             self['NanoCpus'] = nano_cpus
 
+        if runtime:
```

Comment:
What's the minimum API version for this feature? Please add a version check similar to the other options in this method.